addons/web_editor/static/src/js/editor/odoo-editor/src/utils/utils.js

```diff
@@ -1031,9 +1052,16 @@ export const formatSelection = (editor, formatName, {applyStyle, formatProps} =
         let currentNode = selectedTextNode;
         let parentNode = selectedTextNode.parentElement;
 
+        const elHasClassNotFontSizeRelated = el => {
+            return Array.from(el.classList).filter((className) => {
+                return !weUtils.FONT_SIZE_CLASSES.includes(className);
+            }).length > 0;
+        };
```

Comment:
This seems ultra ad-hoc. Why does this function which is very generic needs to know those things, which are very specific ?

Reply:
Because AFAIU, if you have a structure like this:
```html
<span class="xxx">Hello</span>
```
And you select the text "Hello" and execute a command that put another class (yyy) on the text.
```html
<span class="xxx"><span class="yyy">Hello</span></span>
```
That' great for most cases. But for font size,  if xxx and yyy are font size classes, I want to unwrap the element and have something like that:
```html
<span class="yyy">Hello</span>
```